WireUI/Sources/WireMoveToFolderUI/FolderPickerViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public final class FolderPickerViewModel: ObservableObject {
+    @Published private(set) var folders: [Folder] = []
+
+    private let conversation: Conversation
+    private let directory: any FolderDirectoryType
```

Comment:
I don't think this property is needed. Instead you can set folders directly in init without the need for the `loadFolders()` method.

Reply:
I want the folders to be updated every time we show the MoveToFolder View. What if we created a new folder? It wouldn't show if we don't have a property marked as `@Published`